src/components/utils/search-matching-equipments.js

```diff
@@ -21,41 +22,31 @@ export const useSearchMatchingEquipments = (
 ) => {
     const { snackError } = useSnackMessage();
     const [equipmentsFound, setEquipmentsFound] = useState([]);
-    const timer = useRef();
     const lastSearchTermRef = useRef('');
     const { getUseNameParameterKey, getNameOrId } = useNameOrId();
 
     const searchMatchingEquipments = useCallback(
-        (searchTerm, sooner = false) => {
-            clearTimeout(timer.current);
-
-            timer.current = setTimeout(
-                () => {
-                    lastSearchTermRef.current = searchTerm;
-                    searchEquipmentsInfos(
-                        studyUuid,
-                        nodeUuid,
-                        searchTerm,
-                        getUseNameParameterKey,
-                        inUpstreamBuiltParentNode,
-                        equipmentType
-                    )
-                        .then((infos) => {
-                            if (searchTerm === lastSearchTermRef.current) {
-                                setEquipmentsFound(
-                                    makeItems(infos, getNameOrId)
-                                );
-                            } // else ignore results of outdated fetch
-                        })
-                        .catch((error) => {
-                            snackError({
-                                messageTxt: error.message,
-                                headerId: 'equipmentsSearchingError',
-                            });
-                        });
-                },
-                sooner ? 10 : SEARCH_FETCH_TIMEOUT_MILLIS
-            );
+        (searchTerm) => {
```

Comment:
how did you replaced the `sooner = false` parameter usage ?

Reply:
I can't see the utility for the `sooner` param after `debounce` impl for this component

Reply:
ok then